DailyCodingProblem/Program.cs

```diff
@@ -42,5 +44,51 @@ static void Problem1()
 
             Console.WriteLine($"Sorry! There is not two numbers in the list that add up to {number}");
         }
+
+
+        /// <summary>
+        /// Given an array of integers, return a new array such that each element of the new array is the product of all the numbers in the original array except the one at the current index.
+        /// </summary>
+        static void Problem2()
+        {
+            Console.WriteLine("I need you to give me at least five integers. Please enter one at the time!");
```

Comment:
This is fine, but as it's a computer program asking, generally you should avoid using things like "I"  and "Me", and instead just give the command. (i.e. "Please enter at least 5 integers, one at a time.")